site/content/docs/main/file-system-backup.md

```diff
@@ -64,6 +64,11 @@ for namespace2 would be `https://s3-us-west-2.amazonaws.com/bucket/kopia/ns2`.
 There may be additional installation steps depending on the cloud provider plugin you are using. You should refer to the 
 [plugin specific documentation](supported-providers.md) for the must up to date information.  
 
+**Note:** Currently, Velero creates a secret named `velero-repo-credentials` in the velero install namespace, this secret consists of a default backup
+repository password, the backup repository is created during the first execution of FS Backup after installing Velero with node agent. If you do not
+want to use the default password set by Velero, please make sure you update the secret to use your own password of choice, the value of the key to update is `Data[repository-password]`. 
```

Comment:
It may be worthwhile to explicitly state that base64 of the password should be set and not the password itself. Also, it may be better to be more explicit about the side effects of changing password after few backups are done.

Reply:
Done, updated the PR.